libs/transactions/README.md

```diff
@@ -0,0 +1,36 @@
+# @vegaprotocol/transactions
+
+Typescript library for vega transaction types, generated from the [vega proto](https://github.com/vegaprotocol/vega/blob/develop/protos/sources/vega/commands/v1/commands.proto) source where these transactions are defined.
+
+## Dependencies
+
+You will need protobuf installed locally in order to be able to run the generator. You can find the [installation guide here](https://grpc.io/docs/protoc-installation/).
+
+## Usage
+
+You can import the generated types from this libray as:
+
+```
+import { TransactionModel } from '@vegaprotocol/transactions'
+
+type SendBatchMarketInstructions = {
+  transaction: TransactionModel.BatchMarketInstructions
+  ...
+}
+```
+
+## Generating the types
+
+Run `yarn nx run transactions:generate` to regenerate the transaction types from the source proto file.
```

Comment:
**nitpick (non-blocking)**:  It's worth moving the `LOCAL_VEGA_ROOT` stuff above this so ~impatient people~ users don't ~do what I did~ just type this out. Nitpick though, as it's all explained.

Reply:
```bash
WALLET_TRANSACTIONS_ROOT=../vega/protos/sources/ WALLET_TRANSACTIONS_SOURCE=../vega/protos/sources/vega/commands/v1/commands.proto yarn nx run transactions:generate
```

is what I ended up with. I think what's here is fine though - most people won't need to generate it as they're checked in.

Reply:
the reason i didn't do it that way in the suggestions is because I assumed some people like setting env vars in their terminal, so they don't have to manually put the var defs in front of their cmd every time (just like me; sincerely: an impatient  user :) )

Reply:
might actually put both version in. not sure, would that just be confusing, what do you think?

Reply:
gonna leave this how it is, ideally i will want to make a pipeline for this generation when the core gets tagged so nobody needs to run this manually in the future.